src/course-updates/course-update/CourseUpdate.jsx

```diff
@@ -18,6 +20,12 @@ const CourseUpdate = ({
     <div className="course-update" data-testid="course-update">
       <div className="course-update-header">
         <span className="course-update-header__date small font-weight-bold">{dateForUpdate}</span>
+        {!isDateForUpdateValid(dateForUpdate) && (
+          <div className="course-update-header__error">
+            <Icon src={ErrorIcon} alt={intl.formatMessage(messages.errorMessage)} />
+            <span className="message-error small">{intl.formatMessage(messages.errorMessage)}</span>
```

Comment:
Maybe tag `p`? Because it`s an error text description. What do you think?

Reply:
Done